apps/mobile/src/components/action-bar/use-action-bar.ts

```diff
@@ -17,18 +18,22 @@ export function useActionBar() {
   const isActionBarVisible = allowedSegmentConditions.some(condition => condition(segments));
 
   function onAddWallet() {
+    analytics.track('add_wallet_sheet_opened', { source: 'action_bar' });
     addWalletSheetRef.current?.present();
   }
 
   function onOpenSend() {
+    analytics.track('send_sheet_opened', { source: 'action_bar' });
```

Comment:
Do we need to specify source here?

Reply:
We'll likely to add more toggles for global sheets elsewhere in the future.